### DISCUSSION TITLE
how do i copy/moving files using exec?

### DISCUSSION THREAD

**Discussion by nwgat** [sharkdp/fd#1605] [Q&A] [answered]
`fdfind test -exec mv /home/dest/path` dont seem to work

**Comment by tmccombs** [comment, marked as answer]
you would need `fdfind test -exec mv '{}' /home/dest/path`

**Reply by tavianator** [reply]
With GNU coreutils, you could also do `fdfind test -X mv -t /home/dest/path` which should be more efficient

**Reply by nwgat** [reply]
this works great on Raspberry OS (debian) and ubuntu :)
